pkg/gator/expand/expand.go

```diff
@@ -180,15 +185,27 @@ func (er *Expander) add(u *unstructured.Unstructured) error {
 	case isNamespace(u):
 		err = er.addNamespace(u)
 	}
-
 	if err == nil {
 		// Any resource can technically be a generator
 		er.objects = append(er.objects, u)
 	}
 
+	err = er.checkNamespaceSelector(u)
```

Comment:
If we were to do this, we would not check all objects added, just those for which the namespace selector is operationalized (mutators in this case)

Reply:
n/a after dropping option to relax namespace check